core/src/main/java/io/questdb/griffin/engine/functions/groupby/StdDevPopDoubleGroupByFunction.java

```diff
@@ -40,61 +35,18 @@
  *
  * @see <a href="https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Welford's_online_algorithm">Welford's algorithm</a>
```

Comment:
nit: this comment is no longer needed since it's in the parent class. The same suggestion applies to other child classes.